electric.yaml

```
@@ -0,0 +1,113 @@
+package:
+  name: electric
+  version: "1.0.19"
+  epoch: 0
+  description: General-purpose programming language and runtime environment
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - elixir
+      - erlang
+      - erlang-dev
+      - protoc
+  environment:
+    MIX_ENV: "prod"
+    MIX_TARGET: "application"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 543b20446747ebd9431d7175f4838bd13a7e1779
+      repository: https://github.com/electric-sql/electric
+      tag: "@core/sync-service@${{package.version}}"
+
+  - working-directory: "packages/sync-service"
+    runs: |
+      mix local.hex --force && mix local.rebar --force
```

You can specify
```suggestion
      mix local.hex --force && mix local.rebar rebar3 /usr/bin/rebar3
```
We might want to use our rebar3 package from wolfi for this

---

it worked just fine, thanks for the suggestion!